buildSrc/src/main/java/com/google/firebase/gradle/plugins/PomValidator.kt

```diff
@@ -43,10 +43,15 @@ abstract class PomValidator : DefaultTask() {
 
   @TaskAction
   fun run() {
-    var diff = diffWithPomFromURL(getLatestReleasePomUrl())
+    try {
+      var diff = diffWithPomFromURL(getLatestReleasePomUrl())
```

Comment:
I realize I accidentally left this as `var`- this should probably be changed to `val`